src/status_im2/contexts/activity_center/notification/contact_requests/events.cljs

```diff
@@ -0,0 +1,26 @@
+(ns status-im2.contexts.activity-center.notification.contact-requests.events
```

Comment:
Could you elaborate on why you decided to create a new namespace instead of using `status-im2.contexts.activity-center.events`?

Reply:
`status-im2.contexts.activity-center.events` is already quite huge and contains some sort of grouping with comments — for example, there is a section `;;;; Contact verification` there and then all the events that are about contact verification follow. So another option for me was to introduce a separate section  `;;;; Contact requests` and make an already big file even bigger.

Honestly, don't see any point in doing this when events can be placed exactly where they belong and close to where they are actually used. In my opinion, the same should be done with `status-im2.contexts.activity-center.events` to transform it more to the namespace that contains imports of other activity center namespaces + some general events. That will help improving readability.

Reply:
Yeah, I believe most collaborators don't take advantage of collapsible sections in Lisp files. In Emacs it's common to navigate between sections very easily, narrow them in buffers, reorder, etc. And for Lisp development in general it's very common to have bigger namespaces separated by section comments. But yeah, for collaborators that only see those sections as normal comments, the number of lines in the file can be a hassle.

>Honestly, don't see any point in doing this when events can be placed exactly where they belong and close to where they are actually used.

You should consider the history of the file as well. Now that we have a bunch of notifications and new features are being added constantly, the original `events.cljs` file increased considerably in size. And now, it might be better to split into smaller, independent namespaces as you said. The original intention was to avoid having lots of small namespaces, because there were so few event handlers, none to be more precise a couple months ago ;). I hope this clarifies a little bit.

>That will help improving readability.

I agree, for collaborators that don't treat section comments as structural blocks ;) Which is most of the team, so I would welcome this change for sure :100: